Fix link to feathers-hooks-common/#validateschema

metareason opened the pull request:
### Summary

- [x] Tell us about the problem your pull request is solving.
Fix validateSchema link should be going to https://feathers-plus.github.io/v1/feathers-hooks-common/#validateschema
- [x] Are there any open issues that are related to this?
No
- [x] Is this PR dependent on PRs in other repos?
No

eddyystop commented:
An internal reference is much faster for the doc reader to follow than an external one, and an interal one is much more portable also.

I think ../../api/hooks-common#validate-schema instead of ../../api/hooks-common#validate-schema.md would work. Would you mind trying that?

metareason commented:
@eddyystop thanks for reviewing, I checked again and I think the external link to the https://feathers-plus.github.io docs site is correct as there are no docs for validateSchema on this https://generator.feathers-plus.com doc site.

In fact I searched the whole page and found there are actually **three** existing links to `validateSchema` on this page, with the first one already linking to the external site https://feathers-plus.github.io. - I have added a second commit that fixes the third link.

So in total this PR now changes **two** broken links, so **all three** now link to https://feathers-plus.github.io/v1/feathers-hooks-common/index.html#validateSchema

Three validateSchema links are on lines 22, 128, 289 - I modified 128, 289 to be the same as 22.

Can see it here:
https://github.com/feathers-plus/generator-feathers-plus/pull/185/files#diff-c64eb056f17738f6ef71a1db75ef3f4bR22

By the way, thanks for making feathers it is great. I am looking at moving from the standard cli to the feathers-plus version of the cli: the round trip generation, typescript and graphql stuff looks amazing, thus reading your docs while I try it out!

Cheers
